tests/s2n_test.h

```diff
@@ -99,3 +100,5 @@
                                         } \
                                         printf("s2nd entered FIPS mode\n"); \
                                       }
+#define S2N_TEST_ENABLE_TLS13   EXPECT_SUCCESS(setenv("S2N_ENABLE_TLS13_FOR_TESTING", "1", 0)); EXPECT_SUCCESS(s2n_tls13_support_init());
```

Comment:
nit: I think a function is more suitable for this since this action doesn't require any variable state from the current scope.